ai/STA/Tactic/go_kick.py

```diff
@@ -72,20 +67,20 @@ def go_behind_ball(self):
             if self.auto_update_target:
                 self._find_best_passing_option()
         ball_collision = self.tries_flag == 0
-        return MoveToPosition(self.game_state,
-                              self.player,
-                              Pose(distance_behind, orientation),
-                              ball_collision=ball_collision,
-                              cruise_speed=1)
+        return CmdBuilder().withMoveTo(Pose(distance_behind, orientation),
```

Comment:
would be

`AICommand().withMoveTo(Pose(distance_behind, orientation), cruise_speed=1)`